curriculum/challenges/english/25-front-end-development/quiz-javascript-math/66edc3ab8c6413c344f401bf.md

```diff
@@ -17,439 +17,452 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is the result of this code:
+const num1 = Number('888');
+const num2 = new Number('777');
+
+console.log(num1 == 888); //statement 1
+console.log(num2 == 777); // statement 2
 
 #### --distractors--
 
-Placeholder distractor 1
+Both statements are true
 
 ---
 
-Placeholder distractor 2
+Both statements are false
 
 ---
 
-Placeholder distractor 3
+Statement 1 is false while statement 2 is true
 
 #### --answer--
 
-Placeholder answer
+Statement 1 is true while statement 2 is false
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is not an arithmetic operator in JavaScript
 
 #### --distractors--
 
-Placeholder distractor 1
+`%`
 
 ---
 
-Placeholder distractor 2
+`+`
 
 ---
 
-Placeholder distractor 3
+`--`
 
 #### --answer--
 
-Placeholder answer
+`==`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following will log the correct gross profit?
 
 #### --distractors--
 
-Placeholder distractor 1
+`let store1_sales = 7500;
+let store2_sales = 6000;
+console.log("Total Sales: " +  store1_sales + store2_sales);`
```

Comment:
````suggestion
```js
let store1_sales = 7500;
let store2_sales = 6000;
console.log("Total Sales: " +  store1_sales + store2_sales);
```
````
a multiline code snippet requires three backticks above and the lang name, and three backticks below. Please fix the other code snippets